src/FirmwarePlugin/PX4/PX4FirmwarePlugin.cc

```diff
@@ -174,7 +174,7 @@ bool PX4FirmwarePlugin::setFlightMode(const QString& flightMode, uint8_t* base_m
 
 int PX4FirmwarePlugin::manualControlReservedButtonCount(void)
 {
-    return 8;   // 8 buttons reserved for rc switch simulation
+    return 0;   // 0 buttons reserved for rc switch simulation
```

Comment:
@DonLakeFlyer Don't judge debug interfaces you're not using. I have defaulted the simulation to pure joystick, so it should be none of QGC's concern if someone enables RC emulation. And if they do and have buttons dual-assigned its their problem. Lets focus here on the plain joystick support.